### PR TITLE
Inline test should use struct types

### PR DESCRIPTION
**Pull request by TIHan**
I decided to look back at the tests written, realized that I didn't use struct types for two of the types tested. This PR just quickly fixes them and adds another small test.

### CODE REVIEW

**Comment by dsyme** [comment]
Make sense, looks good to me

File: tests/fsharp/optimize/inline/test.fs [hunk 1]
```diff
@@ -12,4 +12,5 @@ let testVector3NestedMutableFieldTestInline (v1: Vector3NestedMutableField) =
     Vector3NestedMutableField.test v1 v1
 
 let testVector3GenericInline (v1: Vector3Generic<'T>) =
-    Vector3Generic.test v1 v1
\ No newline at end of file
+    Vector3Generic.test v1 v1
+    Vector3Generic.testObj
```
Comment (latkin) [inline tests/fsharp/optimize/inline/test.fs]: Was this meant to be left without arguments?

Comment (TIHan) [inline tests/fsharp/optimize/inline/test.fs]: No, it should have arguments. :( Will correct this.

**Comment by latkin** [comment]
:+1:

**Pull request closed without merging**